src/shared/lib/resources/proposal/code-with-us.ts

```diff
@@ -61,12 +75,17 @@ export type CreateProponentRequestBody
 
```

Comment:
I suggest adding a function to `parseCreateProponentRequestBody(raw: any): CreateProponentRequestBody | null`, and calling the function in the back-end CRUD resource when creating.

Reply:
I've implemented this in the back-end resource now (except using a blank individual proponent instead of null as discussed).  Let me know if this works, happy to adjust some more if necessary.